spec/basic/render_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+
+describe "The Page renders", type: :feature, js: true do
+  before :each do
+    visit '/'
+  end
+
+  it "has a page title" do
+    expect(page).to have_content("A Community of Aspiring & Experienced Ruby Developers in Belfast")
+  end
+
+  it "has a header" do 
```

Comment:
Community Standards - if there is no variable name then we favour using a single quote:

"Use doubles if you have to do this, ${name}"

or

'Use single if you have to this, Matt'